src/Core/Domain/MerchandiseReturn/ValueObject/MerchandiseReturnId.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * 2007-2020 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Domain\MerchandiseReturn\ValueObject;
+
+use PrestaShop\PrestaShop\Core\Domain\MerchandiseReturn\Exception\MerchandiseReturnConstraintException;
+
+/**
+ * Provides merchandise return id
+ */
+class MerchandiseReturnId
+{
+    /**
+     * @var int
+     */
+    private $id;
```

Comment:
I know that we have some of other objects similarly implemented, but shouldn't we name the property $id -> $value instead?
Even getter is getValue() but it retrieves the $id, its just abit strange. Lets just leave the property named $value then?

Reply:
I think it is important to have a function to retrieve as getValue so that all those one var VO's behave the same

Other than that having variable as $id makes sense because once open up this VO you clearly see that variable you are passing is an $id and not more ambiguous $value. Sort of if you ever wonder what the value of MerchandiseReturnid is you open function and clearly see it's an id. 

I think both ways work, however since so far everywhere variable always was an $id or some variation of that, I just don't see how we would benefit from changing that.

Reply:
> I think both ways work, however since so far everywhere variable always was an $id or some variation of that, I just don't see how we would benefit from changing that.

I like this pragmatic approach 👍